oidc_example/op2/requirements.txt

```diff
@@ -0,0 +1,19 @@
+alabaster==0.7.12
```

Comment:
I am not a fan of having the dependencies pinned in requirements file. These should only list first level dependencies without specific versions.

The specific versions should rather be in constraints file.

Reply:
Interesting concept, have not tired the constraints file before. :) I gave it a stab in my fix-remarks commit here: 81d56dc

Let me know what you think!

Reply:
Yes, this is exactly what I want to see. The difference here is that the constraints are only applied when you want and also they do not force the dependencies to be installed, so if one of the dependencies drops something, we no longer force the installation of it.